### ISSUE TITLE
Missing "value" field in Highlight when HighlightFullFields is used

### ISSUE THREAD

**Issue by solvingproblemswithtechnology** [DAXGRID/typesense-dotnet#172]
Hi there!

Just missing the value field as per [this github issue](https://github.com/typesense/typesense/issues/106).

An example of the Highlight json with this field:

```json
{
  "field": "title",
  "matched_tokens": [
    "Lorem",
    "Ipsum"
  ],
  "snippet": "<marked>Lorem ipsum</marked> dolor sit amet, consectetur adipiscing elit",
  "value": "<marked>Lorem ipsum</marked> dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. "
}
```

**Comment by runeanielsen** [comment]
Hi @solvingproblemswithtechnology,

I don't see the `value` field mentioned in the post, maybe I missed it, can you clarify where it is described - I checked the documentation, but could not find it in the examples. 

// Rune

**Comment by runeanielsen** [comment]
@solvingproblemswithtechnology makes sense, I'll take a look at it today or tomorrow.

**Comment by runeanielsen** [comment]
@solvingproblemswithtechnology this has now been released in version 6.10.0.

**Comment by solvingproblemswithtechnology** [comment]
Amazing! Thank you!
